lib/checks/checks.go

```diff
@@ -109,9 +109,9 @@ type ManifestValidator struct {
 
 // RunBasicChecks executes a set of additional health checks.
 // Returns list of failed health probes.
-func RunBasicChecks(ctx context.Context, options *validationpb.ValidateOptions) (failed []*agentpb.Probe) {
+func RunBasicChecks(ctx context.Context, options *validationpb.ValidateOptions, openEBSEnabled bool) (failed []*agentpb.Probe) {
```

Comment:
Can this parameter be propagated via existing `ValidateOptions`?

Reply:
> Can this parameter be propagated via existing `ValidateOptions`?

The problem is that there is a call with nil for ValidateOptions:

```
failed = checks.RunBasicChecks(ctx, nil)
```

And RunBasicChecks invokes:

```
		monitoring.DefaultProcessChecker(openEBSEnabled),
```

The above expects a bool and in this case I don't have the needed information if the propagation happens via ValidateOptions.